2/contrib/openshift/configuration/credentials.xml.tpl

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<com.cloudbees.plugins.credentials.SystemCredentialsProvider plugin="credentials@1.23">
+  <domainCredentialsMap class="hudson.util.CopyOnWriteMap$Hash">
+    ${KUBERNETES_CREDENTIALS}
+  </domainCredentialsMap>
+</com.cloudbees.plugins.credentials.SystemCredentialsProvider>
```

Comment:
symlink (with understanding, maybe even a comment hinting at, that the symlink might have to be broken if credentials version diverges over time)